install_config/install/advanced_install.adoc

```diff
@@ -617,6 +620,26 @@ value; you only need to set this if you want your `git clone` operations to use
 a different value.
 |===
 
+[[advanced-install-configuring-firewalls]]
+=== Configuring the Firewall
+
+{product-title} uses iptables as the default firewall, but you can switch to use firewalld in the install.
+
+Because iptables is the default filewall, {product-title} is designed to have it
```

Comment:
@knobunc I think I understand what you're saying now. "Even though iptables is the default, firewalld is recommended, but OpenShift still uses iptables even though firewalld is recommended, so don't have mixed clusters." I think it was words like "previous default" and "old default" that put me off... I added a admonition to the start of the section:

"If you are changing the default firewall, ensure that each host in your cluster is using the same firewall type to prevent inconsistencies."

I think anything else would verge on saying something that's already there.